### ISSUE TITLE
Creating an Account Alias causes error

### ISSUE THREAD

**Issue by MScottBlake** [mastodon/mastodon#19758]
### Steps to reproduce the problem

1. Open Preferences > Account > Account Preferences
2. Click the `create an account alias` link

### Expected behaviour

A form should appear allowing you to create an alias.

### Actual behaviour

A generic error appears on screen

### Detailed description

When you click the `create an account alias` link in Preferences, I am taken to an error page with nothing but this image on it.
<img width="539" alt="image" src="https://user-images.githubusercontent.com/945258/200095792-e7990e8f-d32c-4411-b8fb-b77b5a2d38e0.png">

### Specifications

macOS Monterey 12.6.1 (21G217)
Google Chrome 107.0.5304.87 (Official Build) (x86_64)

**Comment by ClearlyClaire** [comment]
Hi! Thank you for your report.

Have you ever created an alias previously? Which version of Mastodon is your server running?

**Comment by MScottBlake** [comment]
I might have created an alias on this server already, but I'm not sure.
This server is v3.5.0.

I migrated from Twitter last week onto mastodon.lol which was (apparently)
very unceremoniously shut down. I have been trying to figure out a way to
get data from that server to no avail. It's possible that I created an
alias from that server and since it can't be reached, the new server is
failing. I honestly wasn't paying enough attention to know what triggered
this error.
